composer.json

```diff
@@ -408,6 +408,7 @@
             "npm run stylelint-themes"
         ],
         "va:test:accessibility": [
+            "apt-get install chromium",
```

Comment:
Move this to the tugboat `config.yml` init hook.

Reply:
yeah, I [did that initially](https://github.com/department-of-veterans-affairs/va.gov-cms-test/pull/183/commits/f1184eba4aea78cc34bb1b59f8d2ce911f274e0a), just moved it here for faster turnaround while trying to figure out what worked and what didn't